### ISSUE TITLE
Mirror-drawing to the top left after creating a new image

### ISSUE THREAD

**Issue by mrfoogles** [rgab1508/PixelCraft#43]
https://user-images.githubusercontent.com/48422925/104823214-7cf97e00-57fd-11eb-9d86-6f32ce7c41af.mov

**Comment by theabbie** [comment]
@mrfoogles Which version are you using?

Is it https://pixelcraft.web.app ?

**Comment by mrfoogles** [comment]
It happened on the github.io version and a local version I hosted with python http.server.  You can't create a new image on https://pixelcraft.web.app; the button isn't there.
<img width="187" alt="Screenshot" src="https://user-images.githubusercontent.com/48422925/104950000-c02a2d00-5974-11eb-9bae-1b9b23ebabf3.png">

**Comment by theabbie** [comment]
@mrfoogles Yeah, pixelcraft.web.app has the stable version without the save progress feature, It's still in beta so not released on production website. Once we fix it, it will be available.
